homeassistant/components/southern_company/translations/en.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
We don't keep resulting translations in our repository anymore, please remove this file.

Reply:
I assume I remove the entire translations folder? or just en.json?

Reply:
In `git` that will result in the same thing... as git tracks files, not folders. You can't add empty folders :)